README.md

```diff
@@ -6,7 +6,7 @@ An open-source Ethereum consensus client, written in Rust and maintained by Sigm
 
 [Chat Badge]: https://img.shields.io/badge/chat-discord-%237289da
 [Chat Link]: https://discord.gg/cyAszAh
-[Book Status]:https://img.shields.io/badge/user--docs-unstable-informational
+[Book Status]:https://img.shields.io/badge/user--docs-stable-informational
```

Comment:
We're still building the book off `unstable` branch right? 

https://github.com/sigp/lighthouse/blob/8cd2b1ca87a7bf660130279558ed5dd9e91dda75/.github/workflows/book.yml#L6

Reply:
yeah agree, should revert this change